src/Microsoft.Health.Fhir.Core/Features/Definition/SearchParameterDefinitionManager.cs

```diff
@@ -46,22 +45,16 @@ public SearchParameterDefinitionManager(IModelInfoProvider modelInfoProvider)
 
         public void Start()
         {
-            // This method is idempotent because dependent Start methods are not guaranteed to be executed in order.
-            if (!_started)
-            {
-                var builder = new SearchParameterDefinitionBuilder(
-                    _modelInfoProvider,
-                    "search-parameters.json");
-
-                builder.Build();
+            var builder = new SearchParameterDefinitionBuilder(
+                _modelInfoProvider,
+                "search-parameters.json");
 
-                _typeLookup = builder.ResourceTypeDictionary;
-                UrlLookup = builder.UriDictionary;
+            builder.Build();
 
-                List<string> list = UrlLookup.Values.Where(p => p.Type == ValueSets.SearchParamType.Composite).Select(p => string.Join("|", p.Component.Select(c => UrlLookup[c.DefinitionUrl].Type))).Distinct().ToList();
+            _typeLookup = builder.ResourceTypeDictionary;
+            UrlLookup = builder.UriDictionary;
 
-                _started = true;
-            }
+            List<string> list = UrlLookup.Values.Where(p => p.Type == ValueSets.SearchParamType.Composite).Select(p => string.Join("|", p.Component.Select(c => UrlLookup[c.DefinitionUrl].Type))).Distinct().ToList();
```

Comment:
>list [](start = 25, length = 4)

Isn't this variable basically dead and do nothing? Can it be removed?

Reply:
Good question.. @brendankowitz do you know the answer to this?

Reply:
It's dead code. Looks like something I added a year and a half ago in https://github.com/microsoft/fhir-server/commit/38b2510#diff-3aac7071615416b3d949f4eb8b32dadbe3e53ec9c7164378f9aa70d20e7ca7c7R51 for debugging purposes that I forgot to remove 👎  
Thanks for spotting.